task-tracker-webapp/src/main/java/com/isa/tasktrackerwebapp/controller/MainController.java

```diff
@@ -1,12 +1,40 @@
 package com.isa.tasktrackerwebapp.controller;
 
+import com.isa.tasktrackerwebapp.model.PageTitle;
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 @Controller
 public class MainController {
     @GetMapping("/")
-    public String index(){
-    return "index";
+    String index(Model model) {
+        model.addAttribute("content", "index")
+                .addAttribute("pageTitle", PageTitle.INDEX.getDisplayValue());
+        return "main";
+    }
+
+    @GetMapping("/registration")
+    String registration(Model model) {
+        model.addAttribute("content", "registration")
+                .addAttribute("pageTitle", PageTitle.REGISTRATION.getDisplayValue());
+        return "main";
+    }
+
+    @GetMapping("/addtask")
+    String newTask(Model model) {
+        model.addAttribute("content", "addtask")
```

Comment:
jak te `pageTitle` dodajesz z enuma to może `content` też się uda; w sumie enum też może się nazywać po prostu `Page` albo `PageType`, albo coś w tym stylu, bo wtedy nie będzie dotyczył tylko tytułów